packages/components/package.json

```diff
@@ -35,6 +35,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@kaizen/component-base": "^1.1.5",
```

Comment:
We don't use `component-base` (used for `OverrideClassName`) in AIO. This has been copied in and is aliased, so you'll update the import instead to be from `~types/OverrideClassName` :)